client/tasks/task-list-item.tsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import { __, _n } from '@wordpress/i18n';
```

Comment:
Looks like `_n` is unused as well.

```suggestion
import { __ } from '@wordpress/i18n';
```

Reply:
Nice!  I think I'm relying on our linters to catch stuff too much which appear to have taken an extended vacation.